src/index.js

```diff
@@ -20,13 +20,12 @@ if (/^6\./.test(babel.version)) {
 }
 
 const { version } = require("../package.json");
-const cache = require("./cache");
```

Comment:
The `./cache` is now unused, we can remove this file and the only dependency `find-up` after tests are fixed.